app/assets/stylesheets/views/_landing_page/main-navigation.scss

```diff
@@ -0,0 +1,30 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-b-main-navigation {
+  padding-top: govuk-spacing(3);
+  padding-bottom: govuk-spacing(6);
+  padding-left: 0;
+  padding-right: 0;
+  margin: 0;
+
+  .app-b-main-navigation__listitem:last-of-type {
+    margin-right: 0;
+  }
+}
+
+.app-b-main-navigation__listitem {
+  list-style: none;
+  display: inline-block;
+  margin-right: govuk-spacing(9);
+  margin-top: govuk-spacing(3);
+  @include govuk-font(19, $weight: normal);
+
+  @include govuk-media-query($until: "desktop") {
+    display: block;
```

Comment:
Sorry, I've just noticed that the designs have the mobile version as what looks like a dropdown - might be worth checking with design on how this is supposed to work (in the design the active menu item is separate from the dropdown for some reason).

![Screenshot 2024-10-09 at 12 52 12](https://github.com/user-attachments/assets/f0f76868-75c8-4d84-be3b-e2b03eb0f4ef)

Reply:
Obviously this makes this block much more complicated, apologies.

Reply:
@andysellick No worries - thanks for spotting this. I think it's probably easier to have two blocks here, one being the existing desktop/tablet simple menu, and then separately I'll create the more complex mobile one, and show/hide each using our CSS breakpoints. Let me know if you disagree. Will investigate and update this PR once done.

Reply:
I get that duplicating it might be easier but duplicating stuff makes for more stuff to maintain. If you can keep it as one thing I think that'd be better, but obviously if it's too hard for now we could duplicate and fix later.

Starting to think we should have a list of tech debt...